src/components/dapp-staking/statistics/CompoundReward.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div v-if="isSupported">
+    <div class="title">
+      {{ $t('dappStaking.autoCompound') }}
+      <IconTooltip>
+        {{ $t('dappStaking.autoCompoundingTooltip') }}
+      </IconTooltip>
+    </div>
+    <div>
+      <span class="text--title">{{ isCompounding ? 'ON' : 'OFF' }}</span>
+      <Button :small="true" :primary="true" class="button" @click="changeDestination">
+        {{ isCompounding ? 'Turn off' : 'Turn on' }}
```

Comment:
Please use i18n

Reply:
> Please use i18n

@hoonsubin this has been fixed. Please check.